### ISSUE TITLE
[Feature]: Decoy app Icon

### ISSUE THREAD

**Issue by jermanuts** [simplex-chat/simplex-chat#4276]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Platform

Android

### Feature

Something like what is suggested here https://github.com/oxen-io/session-android/issues/1471

![image](https://github.com/simplex-chat/simplex-chat/assets/109705802/646b3615-3aeb-48c1-a347-be4af27c7ea9)

It will be still shown in Apps list in settings tho, so only works in app drawer and home screen.

**Comment by jermanuts** [comment]
>A calculator vault style approach would likely work better.

Like https://discuss.grapheneos.org/d/12681-stealth-calculator-app-hide-apps

But Daniel (GrapheneOS dev) called it also security theatre when it was used in the ANOM operation.

**Comment by greensheeps** [comment]
> > A calculator vault style approach would likely work better.
> 
> Like https://discuss.grapheneos.org/d/12681-stealth-calculator-app-hide-apps
> 
> But Daniel (GrapheneOS dev) called it also security theatre when it was used in the ANOM operation.

I agree, but itt's still much better than Signal's approach which entirely relies on the adversary not opening the app, so if it were to be done, a calculator vault would be the best way.

**Comment by kimg45** [comment]
Not sure if this is a feature in Android but iOS is getting the ability to hide apps in a special folder behind PIN/biometrics. Seems like the OS takes care of this a lot better than the app itself could.

**Comment by greensheeps** [comment]
> Not sure if this is a feature in Android but iOS is getting the ability to hide apps in a special folder behind PIN/biometrics. Seems like the OS takes care of this a lot better than the app itself could.

This is coming in Android 15 https://www.androidcentral.com/apps-software/android-15-private-space-full-details

**Comment by greensheeps** [comment]
I think both of these features can put people at risk. Someone who knows about the app can figure out it's there and you're hiding it, you can just be forced to unlock private space making it not so private... I expect GrapheneOS will extend their [duress PIN](https://grapheneos.org/features#duress) feature to Private Space (that's if they include it at all)

**Comment by greensheeps** [comment]
In it's current state, Private Space seems useful for if you want to hand someone your phone and not see or have access to your private apps, I don't think it's great if you're under duress.

**Comment by kimg45** [comment]
I don't really see how an adversary that is going to scour your phone for all your hidden apps isn't going to find one that just has a different logo or is hidden as a calculator. The police always seem to find those pretty easily.

**Comment by greensheeps** [comment]
> I don't really see how an adversary that is going to scour your phone for all your hidden apps isn't going to find one that just has a different logo or is hidden as a calculator. The police always seem to find those pretty easily.

Yes, I agree. I think Private Space + duress PIN is better, although SimpleX could add the calculator and vault and make the self-destruct passcode work with that also.
